rpmconf/Install/zmsetup.pl

```diff
@@ -7841,6 +7867,36 @@ sub addJDK17Options {
 	}
 }
 
+sub activateLicense {
+	if (isNetwork() && isEnabled("zimbra-store")) {
+		progress ("Looking for valid license to activate...");
+		my $licensecode = $config{LICENSECODE};
+		my $rc = runAsZimbra("/opt/zimbra/bin/zmlicense -c");
+		if ($rc == 256) {
```

Comment:
whats 256 check here?

Reply:
it is the return value given by runAsZimbra function.